test/plugins_test.rb

```diff
@@ -0,0 +1,52 @@
+# -*- encoding : utf-8 -*-
+require 'coveralls'
+Coveralls.wear!
+
+require 'minitest/autorun'
+require 'ffaker'
+
+# Loads lolcommits directly from the lib folder so don't have to create
+# a gem before testing
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
+$LOAD_PATH.unshift(File.dirname(__FILE__))
+require 'lolcommits'
+
+include Lolcommits
+
+class PluginsTest < MiniTest::Test
+  #
+  # issue #57, https://github.com/mroth/lolcommits/issues/57
+  #
+  def test_tranzlate
+    [['what the hell', '(WH|W)UT TEH HELL'], ['seriously wtf', 'SRSLEH WTF']].each do |normal, lol|
+      tranzlated = Lolcommits::Tranzlate.tranzlate(normal)
+      assert_match(/^#{lol}/, tranzlated)
+    end
+  end
+
+  #
+  # issue #136, https://github.com/mroth/lolcommits/issues/136
+  def test_lol_twitter_build_tweet
+    long_commit_message = Faker::Lorem.sentence(500)
+    plugin              = Lolcommits::LolTwitter.new(nil)
+    max_tweet_size      = 116
+    suffix              = '... #lolcommits'
+
```

Comment:
Thats true, these tests will sit outside the main gem in the end, lets punt on this for now.